I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml

```diff
@@ -2189,15 +2390,106 @@ Line |     .  +-----+ | |  *  *  | | +-----+  .     | Line
    
    The optical impairments exposed for this 1+1 protected TE-link are
    typically based on the optical impairments of the worse of the two
-   underlying physical OMS links.
+   underlying physical OMS links including the optical impairments
+   imposed by the splitter (outgoing direction) and selector (incoming
+   direction).
   </t>
-  
+
+  <t>
+   <xref target="OMS_MCG_prot_link_2a"/> illustrates this modeling
+   approach where the splitter/selector in the adjacent WDM-TE-Nodes,
+   WDM-TE-Node-1 and WDM-TE-Node-2, as well as the two physical OMS MCG
+   links are abstracted into a single 1+1 protected TE-link. This is
```

Comment:
What are `two physical OMS MCG links` supposed to describe? is it the MCG in forward and the reverse direction (2 x unidirectional MCGs)?
`ITU-T [G.807] defines the Media Channel Group MCG as
   "A unidirectional point to point management/control abstraction 
   that represents a set of one or more media channels that are 
   co-routed."`

Reply:
Not sure if it is the term "physical" from which this concern was born , but for the rest OMS MCG link meaning are clearly clarified the scope in sec. 8.4.1 of G.872 (12/19) the last one approved. What defined before for OMS link is valid now for this new terminology.

Reply:
In our document we define MCG in section 2.2 as
`According to [G.807], a Media Channel Group (MCG) represents "a
   **unidirectional** point-to-point management/control abstraction that
   represents a set of one or more media channels that are co-routed.  A
   media channel group (MCG) is bounded by a pair of media ports."`

we then define OMS MCG in section 2.3 as:
`
   [G.807] defines an OMS MCG as "The topological relationship between
   the media port on a filter or coupler where a set of media channels
   are aggregated and the media port on a filter or coupler where one or
   more media channel is added to or removed from that aggregate.  All
   of the media channels that are represented by the OMS MCG must be
   carried over the same serial concatenation of OTS MCGs and
   amplifiers."`

Both are logical concepts and are unidirectional. If those are abstracted in a 1+1 protected TE-link there would be 4 OMS MCG to form a bidirectional 1+1 TE-link.

It is indeed unclear what a "physical" OMS MCG is supposed to represent.

Reply:
Not clear what you want here. I've pointed out the reference note in G.872 to explain that what was valid before for OMS link is not represented by OMS MCG new terminology. 
We have an issue related to terminology alignment and we fixed that, btw the text in the section you mention are provided by you. 
If there is something further not clear in the text . the better solution is to provide yourself a text proposal taking into account the deadline for update of July 10. About the physical again , I'm personally free to change wording if all authors/contributors consider "physical" term not clear enough.

Reply:
The word "physical" can be removed if that is creating confusion and if it is causing an inconsistency with the term "OMS MCG link". The intention was to emphasize that the physical links forming the 1+1 protection group are modeled as a single 1+1 protected TE-link in IETF TE modeling language.

Reply:
@sergiobelotti which aspect of G.872 creates an issue here? An OMS is unidirectional and to for a 1+1 bidirectional link one needs 4 of them. Is that something controversial?